device/bgp/bgproutedistinguisher.yaml

```diff
@@ -0,0 +1,24 @@
+components:
+  schemas:
+    Bgp.RouteDistinguisher:
+      description: >-
+        BGP Route Distinguisher.
+      type: object
+      properties:
+        rd_type:
+          description: |-
+            Route Distinguisher Type field of 2 Byte.
```

Comment:
As per @kamalsahu0001 use case, he needs option for auto-configure RD IP Address.  

![image](https://user-images.githubusercontent.com/48308607/165465849-568c179c-f8e1-4118-a8c9-d108424590c1.png)

Reply:
changed `auto_config_rd_ip_addr` default to `false`